assets/css/settings-styles.css

```diff
@@ -1152,6 +1152,81 @@ div.upgrade-table-description a,
 	color: #fcfbf7;
 }
 
+.card-container {
+    border-radius: 5px;
+    width: calc(100% - 30px);
+    display: flex;
+    flex-wrap: wrap;
+    gap: 30px;
+    padding: 15px;
+    background-color: #f0f0f0;
+	justify-content: center;
```

Comment:
I would let the cards left align.

Reply:
@Terminator-Barbapapa I agree. Just FYI, I have placed the justify-content: center on mobile phones media query because there, plugins will be stacked one under another, so it looks weird if they are aligned left.